test/test.js

```diff
@@ -55,19 +62,37 @@ describe('Service', function() {
       var token = {
         access_token: 'foo'
       };
-      var key = service.getPropertyKey_();
+      var key = 'oauth2.test';
       properties.setProperty(key, JSON.stringify(token));
-
-
-      assert.notExists(service.token_);
       assert.deepEqual(service.getToken(), token);
-      assert.deepEqual(service.token_, token);
       assert.deepEqual(JSON.parse(cache.get(key)), token);
     });
+
+    it('should not hit the cache or properties on subsequent calls', function() {
+      var cache = new MockCache();
+      var properties = new MockProperties();
+      var service = OAuth2.createService('test')
+        .setPropertyStore(properties)
```

Comment:
(Again) Usually 4 spaces for continuation.

Reply:
Done